discovery/triton/triton.go

```diff
@@ -54,6 +55,7 @@ var DefaultSDConfig = SDConfig{
 // SDConfig is the configuration for Triton based service discovery.
 type SDConfig struct {
 	Account         string                `yaml:"account"`
+	Role            string                `yaml:"role,omitempty"`
```

Comment:
Given the default is non-empty, we probably don't want the omittempty here as if it ever applied it'd likely just cause confusion.

Reply:
I changed this when falling back to the default config seemed to be broken, but I can't reproduce this now. Thinking this over again I don't think it actually makes a difference for loading yaml files as the field will be set to null anyway.
I'll remove the flag as it's pointless.